package.json

```diff
@@ -4,22 +4,33 @@
   "description": "Federated SAMLP single sign-out",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "./node_modules/.bin/_mocha -R spec --colors",
+    "cover": "nyc npm run test"
   },
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/auth0/samlp-logout.git"
+  },
+  "bugs": {
+    "url": "https://github.com/auth0/samlp-logout/issues"
+  },
+  "author": "Auth0",
   "license": "MIT",
   "dependencies": {
     "@auth0/thumbprint": "0.0.6",
+    "auth0-id-generator": "^0.2.0",
     "ejs": "^2.5.6",
-    "xml-crypto": "~0.10.1",
+    "xml-crypto": "^2.0.0",
```

Comment:
Does this contain breaking changes for consumers of this library?

Reply:
Updated PR description to detail xml-crypto library fixes. The version we were using (0.10.1) unfortunately doesn't have a tag for comparison, but there has been changes for xml c14n - added in 2017.